api/services/sandbox/v1/sandbox.proto

```diff
@@ -96,18 +96,20 @@ service Controller {
 }
 
 message ControllerCreateRequest {
-	string sandbox_id = 1;
-	repeated containerd.types.Mount rootfs = 2;
-	google.protobuf.Any options = 3;
-	string netns_path = 4;
+	string sandboxer = 1;
```

Comment:
Since this API has already been included in a release, we don't want to make numeric changes like this. We are not planning on changing the API versions (from v1 to v2) for the 2.0 release. The sandboxer could be set to a consistent number such as 10 at the end though, the order isn't as important as the compatibility.

Reply:
done